src/sysinfo/nvml.cpp

```diff
@@ -0,0 +1,83 @@
+#include "nvml.h"
+#include "../util/util_string.h"
+#include "../util/util_log.h"
+
+namespace dxvk {
+    Nvml::Nvml() {
+        const auto nvmlModuleName = "nvml.dll";
+        m_nvmlModule = ::LoadLibraryA(nvmlModuleName);
+        if (m_nvmlModule == nullptr) {
+            if (::GetLastError() != ERROR_MOD_NOT_FOUND) // Ignore library not found
+                log::write(str::format("Loading ", nvmlModuleName, " failed with error code ", ::GetLastError()));
+
+            return;
+        }
+
+        nvmlInit_v2 =
+            reinterpret_cast<PFN_nvmlInit_v2>(
+                reinterpret_cast<void*>(
+                    ::GetProcAddress(m_nvmlModule, "nvmlInit_v2")));
```

Comment:
I'd suggest checking if none of requested function pointers is `nullptr` before proceeding with `nvmlInit` call.

Reply:
Good idea, done.